src/pages/iou/IOUDetailsModal.js

```diff
@@ -86,6 +86,10 @@ const defaultProps = {
 
 class IOUDetailsModal extends Component {
     componentDidMount() {
+        if (this.props.network.isOffline) {
+            return;
+        }
+
         this.fetchData();
```

Comment:
Oh wait, won't this prevent `his.fetchData();` from being cached and retried when online again?

Reply:
Tested and yeah from offline to online, will try to fetch again.